contents/docs/data/annotations.mdx

```diff
@@ -8,9 +8,9 @@ availability:
     enterprise: full
 ---
 
-Annotations allow you to leave notes on your charts that can help you understand the driving factors behind the changes that happen over time.
+Annotations allow you to mark when certain changes happened so you can easily see how they impacted your metrics.
```

Comment:
> add context to a point in time on your insights and dashboards

I feel like this is what is confusing. What does it mean to "add context to a point in time"? IMO we need to use more basic language to explain this.

Reply:
Hmm, fair point. I just want to avoid repeating ourselves with "context for changes" across both sentences. Annotations aren't just changes either, they could be used for any type of context (like commentary, bugs). 

Maybe:
- Annotations enable you to add a note to a time on your insights and dashboards...
- Annotations enable you to add a written note with context to a time on your insights and dashboards...

Reply:
Maybe...

Annotations enable you to add written notes to a particular day and time on all your insights and dashboards so you can see how certain changes or happenings impact your metrics.

?